src/praisonai/praisonai/cli/commands/kanban.py

```
@@ -0,0 +1,437 @@
+app = typer.Typer(help="Kanban task management")
+
+
+def _get_kanban_store():
+    """Get kanban store instance."""
+    try:
+        from praisonai.kanban.sqlite_store import SQLiteKanbanStore
+        return SQLiteKanbanStore()
+    except ImportError as e:
+        output = get_output_controller()
+        output.print_error(f"Kanban module not available: {e}")
+        raise typer.Exit(4)
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)